app/src/main/java/edu/unikom/herbamedjabar/view/ProfileFragment.kt

```
@@ -21,19 +20,22 @@ import edu.unikom.herbamedjabar.R
+import kotlinx.coroutines.launch
 
 @AndroidEntryPoint
 class ProfileFragment : Fragment() {
 
     private var _binding: FragmentProfileBinding? = null
-    private val binding get() = _binding!!
+    private val binding
+        get() = _binding!!
 
     private val viewModel: ProfileViewModel by viewModels()
     private lateinit var postAdapter: PostAdapter
```

detekt (1.23.8)

[warning] Usages of lateinit should be avoided.

(detekt.potential-bugs.LateinitUsage)